scripts/sandbox.sh

```diff
@@ -5,25 +5,40 @@ error () {
     echo $1
     exit $last_error
 }
+INITIAL_DIR=`pwd`
 
+: ${SPRINTER_REPO:=toumorokoshi}
```

Comment:
can we call this "sprinter_github_user" instead? I would've guessed "sprinter_repo" refers to the repo name itself, and not the account owner.